src/ontology/ro-edit.owl

```diff
@@ -6791,6 +6822,11 @@ binary, and takes no temporal argument.
           develops_from(x,y)")
 AnnotationAssertion(rdfs:label obo:RO_0001902 "relation has no temporal argument"@en)
 
+# Individual: <https://orcid.org/0000-0001-9625-1899> (Bill Duncan)
```

Comment:
i'm surprised this wasn't already in there, @matentzn is this how it should look for the ORCIDIO import?

Reply:
Is this a blocker for approval of the PR?

Reply:
No, sorry for the late reply (i had some personal things to deal with).

Reply:
I'm checking this now; it should be in the orcidio_import.owl, not the edit file. Have you added this manually, @wdduncan?

Reply:
You should request to add your information in orcidio https://github.com/cthoyt/orcidio. Then, when available, we can update the orcidio_import. Make sure to remove the individual from the edit file, please.

Reply:
Ok. This has been done.

Reply:
You must add the individual to the repository https://github.com/cthoyt/orcidio, not in the `orcidio_import.owl` here. This change will be lost when update the `orcidio_import.owl`

Reply:
`orcidio_import.owl` is like a normal import using ODK, the same as importing terms from OBI or GO. The `orcidio_import.owl` is not the same as the repository https://github.com/cthoyt/orcidio

Reply:
Thanks @anitacaron   
My question was regrading whether the @cthoyt `orcidio_import.owl` file was generated using a script or manually curated.

Reply:
@anitacaron I removed myself from `ordidio_import.owl`.  
I think I should be in the `orcidio_import.owl` file as "William D Duncan". I am in the [file](https://github.com/cthoyt/orcidio/blob/c670db35108e0f8c0716d858b0dcba2a2c928b8b/orcidio.owl#L2894) on https://github.com/cthoyt/orcidio.